frontend/src/containers/Profile/ProfileDetail/ProfileDetail.js

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from "react";
+import {
+    Button, Image, Tabs, Tab,
+} from "react-bootstrap";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+// import { connect } from "react-redux";
+
+import { SideBar, Header } from "../../../components";
+import CollectionCard from "../../../components/Collection/CollectionCard/CollectionCard";
+import ReviewCard from "../../../components/Review/ReviewCard/ReviewCard";
+
+import "./ProfileDetail.css";
+import SamplePhoto from "./sample.jpg";
+// import * as actionCreators from "../../../store/actions/index";
+
+class ProfileDetail extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            doIFollow: this.props.thisUser.doIFollow,
+        };
+    }
+
+    render() {
+        const editButton = (
+            <Link to={`/profile/${this.props.thisUser.id}/edit`}>
+                <Button id="editButton">Edit</Button>
+            </Link>
+        );
+        const followButton = <Button id="followButton" onClick={() => this.setState({ doIFollow: true })}>Follow</Button>;
+        const unfollowButton = <Button id="unfollowButton" onClick={() => this.setState({ doIFollow: false })}>Unfollow</Button>;
+        // only one button will be displayed among "edit", "follow", and "unfollow" buttons
+        let buttonDisplayed;
+        if (this.props.currentUserID === this.props.thisUser.id) {
+            buttonDisplayed = editButton;
+        } else if (this.state.doIFollow) {
+            buttonDisplayed = unfollowButton;
+        } else {
+            buttonDisplayed = followButton;
+        }
+
+        const collectionCardsLeft = this.props.thisUserCollections
```

Comment:
Isn't there better way to make cards(left and right)? For now it could be ok, but I think we could do better.

Reply:
Please pull latest master branch! They were changed in recent versions.

Reply:
I'm afraid that I cannot guess the side effect of using two lined cards for items tabs for now. Do you have any questionable point?

Reply:
I just mentioned it in the aspect of code refactoring.

Reply:
I think it could be improved like feedmaker() in Main.js (by Davin). Could I do that work when dealing with code refactoring issue?

Reply:
@kosumin If you do, it would be great. Could you mention this comment in the related issue?